Source/FieldSolver/SpectralSolver/SpectralFieldDataRZ.cpp

```diff
@@ -398,6 +398,8 @@ SpectralFieldDataRZ::ForwardTransform (const int lev,
                                        amrex::MultiFab const & field_mf, int const field_index,
                                        int const i_comp)
 {
+    amrex::ignore_unused(lev);
```

Comment:
Instead of using `ignore_unused`, I would suggest to do
```suggestion
        amrex::LayoutData<amrex::Real>* cost = WarpX::getCosts(lev);
```
as it is done in the [Cartesian version of the this function](https://github.com/ECP-WarpX/WarpX/blob/development/Source/FieldSolver/SpectralSolver/SpectralFieldData.cpp#L132). This allows the load-balancer to properly take into account the cost of the spectral transform in the cost of each box.

Reply:
Sounds good, thank you! I thought we were not doing that intentionally, for some reasons. Happy to make these changes here.

Reply:
Actually, if I add the `cost` variable, then I think it would make sense to actually use it, as it's done in the Cartesian functions (otherwise the variable `cost` becomes the "unused variable", as `lev` was before). Such as here:
https://github.com/ECP-WarpX/WarpX/blob/8d722c5a621e4552d47275db8485f21b6a28be56/Source/FieldSolver/SpectralSolver/SpectralFieldData.cpp#L151-L154
But I don't think I'm the right person to add those load balancing capabilities to the RZ spectral solver. That is why I just wanted to fix the warning on `lev` being unused. What do you think?

Reply:
Unless it's really just about adding calls like the one linked in my previous comment, in which case I guess I could do that. @mrowan137 Do you have any suggestions on this? Would that require extra testing or would that be OK to add calls similar to the one linked above without me knowing much about it?

Reply:
It would be fine to add the cost calculation to the RZ solver. It looks like it be just copying the code from the Cartesian version at the start and end of the MFIter loop.

Reply:
OK, this should be done in https://github.com/ECP-WarpX/WarpX/pull/2071/commits/6fe444fa5f42dbb60d3ca5f2c6a91e08cad11b64.